rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -102,6 +117,7 @@ fn main() -> Result<()> {
         }
     };
 
+    ensure_gh_setup()?;
```

Comment:
NC

I guess this installs gh if it isn't already installed? It would probably be better if this tool were more self-contained, but this is good enough for now.

Reply:
It doesn't install gh, it just tells you to install it if it's not installed